docs/dietpi_optimised_software.md

```diff
@@ -125,7 +125,7 @@
 - [OpenBazaar - Decentralized peer to peer market server using Bitcoin](https://dietpi.com/phpbb/viewtopic.php?p=1796#p1796)  
 - [YaCy - Decentralized open source search engine](https://dietpi.com/phpbb/viewtopic.php?p=6202#p6202)
 
-## [WiFi HotSpot](../software/wifi_hotspot/)
```

Comment:
I would keep the trailing slash, otherwise there is one more redirect. Run the following two curl commands to see the difference (as well the measurable delay when omitting the slash):
```
time curl -IL https://dietpi.com/docs/software/desktop/
time curl -IL https://dietpi.com/docs/software/desktop
```

So this is basically something we could keep in mind and add the slash to all URLs, when editing pages files.

Reply:
Ok   
@fpetru: I will change this in the whole file.